src/functionalTest/java/uk/gov/hmcts/reform/et/syaapi/Role.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.et.syaapi;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Role {
```

Comment:
Again, should be able to use @DaTa, @AllArgsConstructor and @getter here, surely?